Android/app/src/main/res/layout/content_my_trip_info.xml

```diff
@@ -44,48 +43,44 @@
                     android:layout_weight="1"
                     android:fontFamily="sans-serif-bold"
                     android:padding="10dp"
+                    android:text="Chennai"
```

Comment:
don't add default text

Reply:
Done

Reply:
Whichever review comment you have marked as "Done" are not. Please check

Reply:
??